### ISSUE TITLE
Error: Failed to parse python binary

### ISSUE THREAD

**Issue by micln** [benfred/py-spy#405]
```
$ which python
/opt/************/bin/python

$ ps 311
    PID TTY      STAT   TIME COMMAND
    311 ?        Sl    23:59 python server.py

$ py-spy record -o profile.svg --pid  311
Error: Failed to parse python binary
Reason: Malformed entity: Section 106 size (4277248) + addr (120) is out of bounds. Overflowed: false
```

how can i solve this problem? thanks

**Comment by Jongy** [comment]
Related: https://github.com/benfred/py-spy/issues/403

Can you please post the Python version? (`python -c 'import sys; print(sys.version)'`)

**Comment by benfred** [comment]
@micln can you try the latest version of py-spy ? I suspect this is fixed by #407

**Comment by benfred** [comment]
I'm going to close, since I think this is resolved now.  Feel free to re-open with more details if this is still a problem
